src/wallet/test/coinselector_tests.cpp

```diff
@@ -136,6 +141,14 @@ inline std::vector<OutputGroup>& GroupCoins(const std::vector<COutput>& coins)
     return static_groups;
 }
 
+inline std::vector<OutputGroup>& KnapsackGroupOutputs(const CoinEligibilityFilter& filter)
+{
+    static std::vector<OutputGroup> static_groups;
+    static_groups.clear();
```

Comment:
Why `.clear()`? It does not change the capacity (actual internal buffer size, if that was in mind). 

Just next line completely re-assigns `static_groups` and it seems that should be enough?

Reply:
This was copied from one of the other functions, but indeed is unnecessary, so I have removed it.